src/core/api/index.js

```diff
@@ -141,8 +141,14 @@ export function callApi({
     options.credentials = 'include';
   }
   if (body) {
-    options.body = JSON.stringify(body);
-    options.headers['Content-type'] = 'application/json';
+    if (body instanceof FormData) {
+      options.body = body;
+      // Let the browser sets this header, including the boundary value.
+      delete options.headers['Content-type'];
```

Comment:
This is not set at this point (i.e., `headers` is still just `{}`), so do we need to `delete` it?

Reply:
I wanted to make sure we never set this header by accident, even in the future. In addition, it describes exactly what we need here. We should never set a `content-type` here, otherwise the API request will be invalid. Does that make sense or should I just leave a comment?

Reply:
Yes, I suppose that makes sense. Although I maintain that it's not needed. 😉